networks/grpc/gClient.go

```diff
@@ -62,104 +61,6 @@ func NewgKlaytnClient(addr string) (*gKlaytnClient, error) {
 
 const timeout = 5 * time.Minute
 
-// TODO-Klaytn-gRPC Move the test below to gClient_test.go
-func TestAll() {
-	waitGroup := &sync.WaitGroup{}
-	waitGroup.Add(3)
-
-	address := "127.0.0.1:4000"
-
-	go testCall(address, waitGroup)
-
-	go testSubscribe(address, waitGroup)
-
-	go testBiCall(address, waitGroup)
-
-	waitGroup.Wait()
-}
-
-func testCall(addr string, wg *sync.WaitGroup) {
-	defer wg.Done()
-
-	kclient, _ := NewgKlaytnClient(addr)
-	defer kclient.Close()
-
-	knclient, err := kclient.makeKlaytnClient(timeout)
-	if err != nil {
-		fmt.Printf("fail to make klaytnNodeClient: err=%v\n", err)
-		return
-	}
-
-	//params := []interface{}{"klay_blockNumber"}
-	request, err := kclient.makeRPCRequest("klay", "klay_blockNumber", nil)
-	if err != nil {
-		fmt.Printf("fail to make RPCRequest: err=%v\n", err)
-		return
-	}
-
-	response, err := knclient.Call(kclient.ctx, request)
-	if err != nil {
-		fmt.Printf("fail to call: err=%v\n", err)
-		return
-	}
-	if err := kclient.handleRPCResponse(response); err != nil {
-		fmt.Printf("fail to handle RPCResponse: err=%v\n", err)
-		return
-	}
-}
-
-func testSubscribe(addr string, wg *sync.WaitGroup) {
```

Comment:
Why do you want to make separated PR for this?
Any special reason?

Reply:
`testSubscribe` did not work well. I tried to check with the author if there was an issue in the implementation and put the relevant code.